src/python/libcgroup.pyx

```diff
@@ -277,6 +277,31 @@ cdef class Cgroup:
         if ret != 0:
             raise RuntimeError("Failed to create cgroup: {}".format(ret))
 
+    def cgroup_list_mount_points(self, version):
+        """List cgroup mount points of the specified version
+
+        Arguments:
+        version - It specifies the cgroup version
+
+        Description:
+        Parse the /proc/mounts and list the cgroup mount points matching the
+        version
+
+        Note:
+        List cgroup mount points the specified version"""
```

Comment:
This note seems redundant with the main comment.  Would it make sense to remove it?

Reply:
@drakenclimber Thanks for the review. Agree, will remove the redundant comment.